armi/utils/units.py

```diff
@@ -17,26 +17,44 @@
 import scipy.constants
 
 
-# Names
-ANGLE_DEGREES = "degrees"
-ANGLE_RADIANS = "radians"
-CM = "cm"
-DAYS = "days"
-DEGC = chr(176) + "C"
+# Units (misc)
 DPA = "dpa"
-EV = "eV"
-GRAMS = "g"
-KG = "kg"
-METERS = "m"
-MICRONS = chr(181) + "m"
+FIMA = r"%FIMA"
+MB = "MB"  # megabytes
+MOLES = "mole"
 PASCALS = "Pa"
+PCM = "pcm"
 PERCENT = "%"
-SECONDS = "s"
+REACTIVITY = chr(916) + "k/k"
 UNITLESS = ""
+# Units (angles)
+DEGREES = "degrees"
+RADIANS = "radians"
+# Units (energy)
+EV = "eV"
+MW = "MW"
 WATTS = "W"
+# Units (length)
+CM = "cm"
+METERS = "m"
+MICRONS = chr(181) + "m"
+# Units (mass)
+GRAMS = "g"
+KG = "kg"
+MT = "MT"
+# Units (money)
+CENTS = "cents"  # pennies: $0.01
+DOLLARS = "$"
+# Units (temperature)
+DEGC = chr(176) + "C"
+DEGK = "K"
```

Comment:
Sure. But since nearly everyone will be using the metric system, I want to make sure that prevent people from seeing `K` and think "kilo".

Not that it's `DEG = "K"` and not `DEG = "°K" or something.